src/libutil/tarfile.cc

```diff
@@ -106,7 +104,7 @@ TarArchive::TarArchive(Source & source, bool raw, std::optional<std::string> com
         "Failed to open archive (%s)");
 }
 
-TarArchive::TarArchive(const fs::path & path)
+TarArchive::TarArchive(const std::filesystem::path & path)
```

Comment:
Do not want to change this

Reply:
Why not? The header file uses `std::filesystem::path` as well:
```
    explicit TarArchive(const std::filesystem::path & path);
```

Reply:
I do care less in the `cc` file; I just think if we're bothering to make the `fs` shorthand we might as well use it.